src/app/components/wrapper-components/GameWrapper.ts

```diff
@@ -104,9 +105,10 @@ export abstract class GameWrapper<P extends Comparable> {
     }
     public setRole(role: PlayerOrNone): void {
         this.role = role;
-        this.gameComponent.role = this.role;
-        if (this.gameComponent.hasAsymmetricBoard) {
-            this.gameComponent.rotation = 'rotate(' + (this.role.value * 180) + ')';
+        if (role !== PlayerOrNone.ZERO) {
+            this.gameComponent.setPointOfView(role as Player);
+        } else {
+            this.gameComponent.setPointOfView(Player.ZERO);
```

Comment:
prefer if true else to if not true else !

Reply:
ouais, d'ailleurs c'est buggé, ça devait être avec NONE la comparaison !